packages/mobile/src/pincode/Pincode.tsx

```diff
@@ -163,31 +162,35 @@ export class Pincode extends React.Component<Props, State> {
         return (
           <View style={style.pincodeContent}>
             <Text style={[fontStyles.h1, style.h1]}>{t('createPin.title')}</Text>
-            <NumberInput
```

Comment:
Why change this?

Reply:
Originally I was going to merge `NumberInput` into the `ValidatedTextInput` but it seemed like the component wasn't necessary in the first place so I changed it to use `TextInput`.  I guess the change isn't really directly related anymore so I can revert this change if needed.

Reply:
The pin is meant to be a number. Unless you tested this and found it works better, seems like some kind of NumberInput is the better here

Reply:
From what I've found, the `NumberInput` component didn't have any extra functionality other than having `maxLength={6}` and wrapped in `View` with some padding.

While it uses the numeric keyboardType which only allows numbers to be input, it's still possible to input invalid characters via copy-pasting so I've just now replaced it with the `ValidatedTextInput`.